src/api/routes.js

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
The api folder is currently only used for api calls to external services. It would probably be better for this file to be in the root src folder or src/routes/index.js